canvas_modules/common-canvas/src/palette/palette-content-list-item.jsx

```diff
@@ -299,6 +306,8 @@ class PaletteContentListItem extends React.Component {
 				className={mainDivClass}
 				onMouseOver={this.onMouseOver}
 				onMouseLeave={this.onMouseLeave}
+				tabIndex={0}
+				onKeyPress={this.keyPressEvent}
```

Comment:
If you look at the other code here the function name always matches the prop name e.g. 
onMouseOver -> this.onMouseOver
onMouseLeave -> this.onMouseLeave
So why not follow the same convention an use `onKeyPress` for the method name here?

Reply:
keyPressEvent got updated into onKeyPress.